plugins/src/lib.rs

```diff
@@ -215,6 +215,19 @@ impl Parse for DeriveSerde {
     }
 }
 
+/// Generates:
+/// - derive of Debug, serde Serialize & Deserialize
+/// - serde crate annotation
+#[proc_macro_attribute]
+pub fn derive_serde(_attr: TokenStream, item: TokenStream) -> TokenStream {
+    let mut gen: proc_macro2::TokenStream = quote! {
+        #[derive(Debug, tarpc::serde::Serialize, tarpc::serde::Deserialize)]
```

Comment:
tiny nit: I don't think there's any reason to derive Debug here rather than allowing users to derive Debug themselves. The main reason we reexport the serde derives is to avoid versioning conflicts.